src/OSPSuite.R/MinimalImplementations/ExceptionManager.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using OSPSuite.Utility.Exceptions;
+using OSPSuite.Utility.Extensions;
+
+namespace OSPSuite.R.MinimalImplementations
+{
+   public class ExceptionManager : ExceptionManagerBase
+   {
+      public override void LogException(Exception ex)
+      {
+         Console.WriteLine(ex.FullMessage());
```

Comment:
Here _Console_ would be R output window? (or would the error message just disappear?)

Reply:
yes R output